velox/docs/monthly-updates/december-2023.rst

```diff
@@ -0,0 +1,99 @@
+********************
+December 2023 Update
+********************
+
+Documentation
+=============
+
+* Add documentation for :doc:`Runtime Metric</metrics>`.
+
+Core Library
+============
+
+* Add support for spilling during the output processing stage of the ``OrderBy`` operator.
```

Comment:
perhaps, move this next to the other spilling item "Add support for metrics related to memory arbitration and spilling"